ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -43,6 +43,11 @@ def AddV2(*, x, y, name="AddV2"):
     return ivy.addv2(tf_frontend.to_tensor(x), tf_frontend.to_tensor(y))
 
 
+@to_ivy_arrays_and_back
+def Angle(*, inputs, deg: bool | None = False, name="Angle"):
```

Comment:
The arguments should be exactly the same as in the original documentation of the function.